pkg/3scale/amp/component/deployments_lister.go

```diff
@@ -0,0 +1,52 @@
+package component
```

Comment:
I created a new component that returns you the desired deployment names depending on the configuration set.